src/main/java/com/epam/aidial/core/AiDial.java

```diff
@@ -157,11 +156,9 @@ void stop() {
             close(storage);
             close(redis);
             log.info("Proxy stopped");
-            LogConfigurator.unconfigure();
         } catch (Throwable e) {
             log.warn("Proxy failed to stop:", e);
-            LogConfigurator.unconfigure();
-            System.exit(-1);
+            throw e;
```

Comment:
Why did you remove these lines?

Reply:
I moved it to the main method. start/stop should not change global settings and should not have global-like behavior.